src/api/libcellml/units.h

```diff
@@ -385,6 +385,23 @@ class LIBCELLML_EXPORT Units: public NamedEntity, public ImportedEntity
      */
     static double scalingFactor(const UnitsPtr &units1, const UnitsPtr &units2);
 
+    /** 
+	* @brief Finds and updates the multiplier of the unit.
+	* 
+	* We pass in the unit and use its' attributes to find the relevant multiplier.
+	* If the units are not base units, we travel up the model hierarchy to find 
+	* the base units.
+	*
+	* @param multiplier The multiplier to find.
+	* @param units The units to find the multiplier for.
+	* @param uExp The exponential of the units.
+	* @param logMult The log multiplier of the units.
+	* @param direction The direction to update multiplier. Either 1 or -1.
+	*
+	* @return Either @c true or @c false, depending if the units were successfully updated.
+	*/
+    static bool Units::updateUnitMultiplier(double &multiplier, const UnitsPtr &units, double uExp, double logMult, int direction);
```

Comment:
This doesn't belong in the public API.

Reply:
I'll make it private now.

Reply:
Doesn't even need to be private it was just fine being a function in the `units.cpp` implementation unit.